wolfcrypt/src/asn.c

```diff
@@ -16453,9 +16561,11 @@ int wc_EccPublicKeyDecode(const byte* input, word32* inOutIdx,
 
 #if defined(HAVE_ECC_KEY_EXPORT) && !defined(NO_ASN_CRYPT)
 /* build DER formatted ECC key, include optional public key if requested,
+ * if skipcrv is set 1, it skips curve id for write. In the case, curve id
```

Comment:
I think "In the case" should be "In this case", if I understand what this function is doing correctly.

Reply:
Thanks. I am trying to describe what the function does. Additional comments are welcome.